koku/reporting_common/data/source_service_product.json

```diff
@@ -0,0 +1,58 @@
+[
+    {
+        "source": "AWS",
+        "service_category": "database",
+        "product_codes": [
+            "AmazonRDS",
+            "AmazonDynamoDB",
+            "AmazonElastiCache",
+            "AmazonNeptune",
+            "AmazonRedshift",
+            "AmazonDocumentDB"
+        ]
+    },
+    {
+        "source": "AWS",
+        "service_category": "network",
+        "product_codes": [
+            "AmazonVPC",
+            "AmazonCloudFront",
+            "AmazonRoute53",
+            "AmazonAPIGateway"
+        ]
+    },
+    {
+        "source": "AWS",
```

Comment:
The APIs don't currently look at product_codes for compute and storage, so I think this opens up the possibility of returning results inconsistent with the filters on the APIs.